server/resources/UserResources.py

```diff
@@ -81,3 +81,19 @@ def post(self):
             return {'message': 'Something went wrong'}, 500
 
 
+class UserProspects(Resource):
+    @jwt_required
+    def get(self):
+        current_user = UserModel.find_by_id(get_jwt_identity())
+        prospects = []
+        for prospect in current_user.prospects:
```

Comment:
I'd look into seeing if we can get the database to do some of this work

Reply:
Thanks! Let me know after you look into it!